### ISSUE TITLE
Ajouter NavBar

### ISSUE THREAD

**Issue by RaWaser** [eracom-ID402/Rackel_Waser_Crypto_Website#2]
Ajouter une barre de navigation

**Comment by anaissierro** [comment]
Je vais m'en occuper : )

**Comment by anaissierro** [comment]
L'issue a été réalisée

**Comment by RaWaser** [comment]
Parfait, merci !

**Comment by ms-studio** [comment]
Petite question: êtes-vous passé par l'étape "Pull Request" ? Je n'en trouve pas de trace. Ou avez-vous transmis le code à @RaWaser par un autre moyen?

**Comment by anaissierro** [comment]
Oui, je l'ai fait. Je n'ai, en revanche, pas fait de deuxième branche, étant passée à côté de l'information avant de le faire...
